app/src/pages/_app.tsx

```diff
@@ -43,7 +43,8 @@ const MyApp: AppType<{ session: Session | null }> = ({
           <SyncAppStore />
           <Favicon />
           <ChakraThemeProvider>
-            <QueryParamProvider adapter={NextAdapterPages}>
+            {/* enableBatching allows to update 2 states at once. It is crucial for the Usage page. */}
+            <QueryParamProvider adapter={NextAdapterPages} options={{ enableBatching: true }}>
```

Comment:
I've wished we had a way to do this before, nice find!

Reply:
Batching may not be necessary for this PR, but it's good to know we can add this when building future features.